pkgs/os-specific/linux/kernel/manual-config.nix

```diff
@@ -151,7 +171,7 @@ let
         unlink $out/lib/modules/${modDirVersion}/source
 
         mkdir -p $dev/lib/modules/${modDirVersion}/build
-        cp -dpR ../$sourceRoot $dev/lib/modules/${modDirVersion}/source
+        cp -dpR .. $dev/lib/modules/${modDirVersion}/source
```

Comment:
Hmm. Does this now copy way too much to `$dev/lib/modules/${modDirVersion}/source`?

Reply:
This is a problem I realized while testing with nox-review, I moved the build folder to within source (aka someFolder/source/build) so that it looks more like other packages (cmake-based etc). It used to have this architecture:
someFolder
|--build
|--source

To sum up, it should be exactly the same as before. my nox-review has been running for a day but it still hasn't met any error so I without further comments I would mark the PR ready.

Reply:
nevermind looks like I haven't pushed the latest changes. Need to rebase with the new kernel updates on master too :'(